### PR TITLE
[fix] Reenable test return logits

### PR DESCRIPTION
**Pull request by dcampora**
# Reenable test return logits

Solves https://nvbugspro.nvidia.com/bug/5327892

### CODE REVIEW

File: tests/unittest/_torch/test_return_logits.py [hunk 1]
```diff
@@ -102,7 +95,9 @@ def test_generate_with_return_logits(disable_overlap_scheduler: bool,
         for output in llm.generate(prompts, sampling_params=sampling_params):
             if gather_context_logits:
                 assert output.context_logits is not None
-                assert len(prompts[0].split()) == output.context_logits.shape[0]
+                expected_len = len(prompts[0].split(
+                )) if disable_overlap_scheduler else len(prompts[0].split()) + 1
+                assert expected_len == output.context_logits.shape[0]
```
[the remark this answers is not in the document]
Comment (dcampora) [reply tests/unittest/_torch/test_return_logits.py:100]: As you cleared out, this is added by the tokenizer of tinyllama. Added a comment explaining.

**Review by Funatiq — APPROVED** [review]
